test/packages/input_groups/manifest.yml

```diff
@@ -0,0 +1,132 @@
+conditions:
+  kibana.version: '^7.9.0'
+icons:
+  - src: /img/aws-logo.svg
+    title: logo aws
+    size: 32x32
+    type: image/svg+xml
+screenshots:
+  - src: /img/aws-general-dashboard.png
+    title: General AWS dashboard
+    size: 1702x1063
+    type: image/png
+vars:
+  - name: access_key_id
+    type: text
+    title: Access Key ID
+  - name: secret_access_key
+    type: text
+    title: Secret Access Key
+policy_templates:
+  - name: ec2
+    title: AWS EC2
+    description: Collect logs and metrics from EC2 service
+    inputs:
+      - type: s3
+        title: Collect logs from EC2 service
+        description: Collecting EC2 logs using S3 input
+        input_group: logs
+        vars:
+          - name: visibility_timeout
+            type: text
+            title: Visibility Timeout
+            description: The duration that the received messages are hidden from subsequent retrieve requests after being retrieved by a ReceiveMessage request.  The maximum is 12 hours.
+          - name: api_timeout
+            type: text
+            title: API Timeout
+            description: The maximum duration of AWS API can take. The maximum is half of the visibility timeout value.
+      - type: aws/metrics
+        title: Collect metrics from EC2 service
+        description: Collecting EC2 metrics using AWS CloudWatch
+        input_group: metrics
+        vars:
+          - name: endpoint
+            type: text
+            title: Endpoint
+            default: "amazonaws.com"
+            description: URL of the entry point for an AWS web service.
+    icons:
+      - src: /img/aws-ec2.svg
+        title: AWS EC2 logo
+        size: 32x32
+        type: image/svg+xml
+    screenshots:
+      - src: /img/aws-ec2-overview.png
+        title: AWS EC2 Overview
+        size: 1702x1063
+        type: image/png
+    vars:
+      - name: ec2_foobar_property
+        type: bool
+        title: EC2 Specific Property
+  - name: dynamodb
+    title: AWS DynamoDB
+    description: Collect logs and metrics from DynamoDB service
+    categories:
+      - datastore
+    inputs:
+      - type: s3
+        title: Collect logs from the DynamoDB service
+        description: Collecting DynamoDB logs using S3 input
+        input_group: logs
+      - type: aws/metrics
+        title: Collect metrics from DynamoDB service
+        description: Collecting DynamoDB metrics using AWS CloudWatch
+        input_group: metrics
+    icons:
+      - src: /img/aws-dynamodb.svg
+        title: AWS DynamoDB logo
+        size: 32x32
+        type: image/svg+xml
+    screenshots:
+      - src: /img/aws-dynamodb-overview.png
+        title: AWS DynamoDB Overview
+        size: 1702x1063
+        type: image/png
+    vars:
+      - name: dynamodb_foobar_property
+        type: bool
+        title: DynamoDB Specific Property
+  - name: barracuda
+    title: Barracuda logs
+    description: Collect Barracuda logs from syslog or a file.
+    inputs:
+      - type: udp
```

Comment:
Do we pull in here the additional vars from the data_stream manifest.yml? How do we know which one to select? What if there are multiple data_streams which configure `udp`? I remember I asked the question below but don't remember the answer :-(

Reply:
> Do we pull in here the additional vars from the data_stream manifest.yml?

I think so.

Reply:
> How do we know which one to select? What if there are multiple data_streams which configure udp? 

I don't reckon if we came up with a solution. Do you think we should introduce a "policy_templates" field to data stream manifest or do you see other options?

Reply:
Yes, my understanding is also that such a problem (or is it a feature? 😎) already exists. I'm wary of changing this behavior, unless we can keep the _default_ behavior the same as it is today.

Reply:
What do you think about putting this is in a separate PR if this became an issue? We know that it can be solved with an additional field - `policy_templates` in `data_streams`.

Reply:
Yes, I'm ++ to taking this specific discussion to a follow up issue/PR.

Reply:
The problem does not exist today because we only have 1 policy template so we need all inputs. But with multiple policy templates, not all data_streams are needed for each one. So I think we need to solve it here.

Reply:
I'm looking for propositions/ideas. I've mentioned one idea above.

Reply:
I think both ways you proposed are feasible. I'm leaning in the direction of specifying it in the policy template. Let me explain why.

Lets assume I built a policy_template for a use case. Now I decide, I also want to allow users to configure data_stream x. All I need to do, is update my policy template and add an entry for it.

Having it in the policy template also makes it easy to understand, which data streams are used. I don't need to go to each data stream to check if the policy is mentioned in there.

Reply:
I'm afraid it (_specifying it in the policy template_ ) would violate the open-closed principle, but we can turn a blind eye here. Let me try to introduce it.

Reply:
Just a proposal, happy to discuss other solutions :-)

Reply:
I added the `data_streams` property. Actually I'm fine with both solutions, I wouldn't say it's a concept hard to understand :)

Reply:
I'm good with the proposed solution with two comments:

1. **Default behavior**: not specifying `data_streams` in a policy template needs to be interpreted as if all data streams have been specified. This would guarantee backwards compatibility. I assume such default behavior logic will need to go in the Fleet UI code?

2. **Referential integrity**: We'll need a way to validate that data streams specified under `data_streams` actually exist in the package.

Neither comment above blocks this PR; just things to address in follow up PRs.

Reply:
@ycombinator do you have any preference where `data_streams` proporty should be placed, either in `policy_template` or `input`? For reference: https://github.com/elastic/package-spec/commit/ed04c0f3a449872d3f281ea250c921b340a43668#commitcomment-47498856